static/js/status.js

```diff
@@ -0,0 +1,108 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+
+gtag('config', 'UA-72356300-6');
```

Comment:
Why is this being moved? I think this should probably stay where it was. It's best practice to keep this Google Analytics stuff in the `<script>` tag so that it can easily be updated later if the Analytics/gtag stuff ever updates.

Reply:
Sure, sorry about that! I'll move that back. I wasn't sure if you wanted me to move that code as well.